pdc_client/__init__.py

```diff
@@ -56,14 +56,23 @@ def get_version():
 
 def _read_file(file_path):
     data = {}
-    if isfile(file_path):
-        with open(file_path, 'r') as config_file:
-            data = json.load(config_file)
+    if isdir(file_path):
+        files_list = os.listdir(file_path)
+        for file_name in files_list:
+            file_abspath = os.path.join(file_path, file_name)
+            with open(file_abspath, 'r') as config_file:
+                config_dict = json.load(config_file)
+                same_key = set(data.keys()) & set(config_dict.keys())
+                if same_key:
+                    print "Error: '%s' keys existed in both %s config files" % (same_key, files_list)
+                    sys.exit(1)
+                else:
+                    data.update(config_dict)
     return data
 
 
 def read_config_file(server_alias):
-    result = _read_file(GLOBAL_CONFIG_FILE).get(server_alias, {})
+    result = _read_file(GLOBAL_CONFIG_DIR).get(server_alias, {})
     result.update(_read_file(USER_SPECIFIC_CONFIG_FILE).get(server_alias, {}))
```

Comment:
USER_SPECIFIC_CONFIG_FILE is a file, {} will be returned from _read_file, right?